source/ShipInfoDisplay.cpp

```diff
@@ -350,6 +354,23 @@ void ShipInfoDisplay::UpdateAttributes(const Ship &ship, const Depreciation &dep
 		* (1. + attributes.Get("hull heat multiplier")) : 0.;
 	heatTable.push_back(Format::Number(60. * (shieldHeat + hullHeat)));
 
+    // Add up the maximum possible energy and heat changes; add the total to the table.
+    attributesHeight += 20;
+	const double overallEnergy = idleEnergyPerFrame
+		- movingEnergyPerFrame
+		- firingEnergy
+		- shieldEnergy
+		- hullEnergy;
+	const double overallHeat = idleHeatPerFrame
+		+ movingHeatPerFrame
+		+ firingHeat
+		+ shieldHeat
+		+ hullHeat;
+	tableLabels.push_back("overall:");
```

Comment:
shouldnt this be something like `max usage`?

Reply:
Instead of "overall," you mean?

Reply:
yes